sos.spec

```diff
@@ -1,10 +1,10 @@
 Summary: A set of tools to gather troubleshooting information from a system
 Name: sos
 Version: 4.5.1
-Release: 1%{?dist}
+Release: 2%{?dist}
 Group: Applications/System
 Source0: https://github.com/sosreport/sos/archive/%{name}-%{version}.tar.gz
-License: GPLv2+
+License: GPL-2.0-or-later
```

Comment:
Ack to this change, but I don't think we need to cut a new release just for this.